configs/openmrs_config/concepts/physical_examination.csv

```diff
@@ -38,7 +38,7 @@ fcf078a5-a839-49d7-99cf-5427b0fbaad3,,,Acute myocardial infarction,ជំងឺ
 95496c04-b5c7-4dc6-ad0f-664bf47a98be,,,Angina,ចុកទ្រូង,Angina,ចុកទ្រូង,,,Misc,N/A,,,,
 ad681aca-d429-49f0-b390-e1edd3050503,,,Pulmonary Oedema,ហើមសួត,Pulmonary Oedema,ហើមសួត,,,Misc,N/A,,,,
 16a53891-b85e-4914-b6dc-c7ab25cc29f2,,,TIA,គ្រោះថ្នាក់សរសៃឈាមខួរក្បាលទ្រង់ទ្រាយតូច (TIA),TIA,គ្រោះថ្នាក់សរសៃឈាមខួរក្បាលទ្រង់ទ្រាយតូច (TIA),,,Misc,N/A,,,,
-227e22c4-1982-4510-827f-2440a1cdbc56,,,CVA,គ្រោះថ្នាក់សរសៃឈាមខួរក្បាល,CVA,គ្រោះថ្នាក់សរសៃឈាមខួរក្បាល,,,Misc,N/A,,,,
+227e22c4-1982-4510-827f-2440a1cdbc56,,,Cerebrovascular Accident (I64),គ្រោះថ្នាក់សរសៃឈាមខួរក្បាល,Cerebrovascular Accident (I64),គ្រោះថ្នាក់សរសៃឈាមខួរក្បាល,,,Misc,N/A,,,,
 7d614732-0eb1-426e-8ede-a8c93d2d7b41,,,Ischemia,ស្ទះសរសៃឈាមខួរក្បាល,Ischemia,ស្ទះសរសៃឈាមខួរក្បាល,,,Misc,N/A,,,,
 880a9a97-0885-4bc0-a421-f4edaafc2f5c,,,Hemorrhagic,ដាច់សរសៃឈាមខួរក្បាល,Hemorrhagic,ដាច់សរសៃឈាមខួរក្បាល,,,Misc,N/A,,,,
 5969b812-49bf-48d1-8477-e33935141c0c,,,Cardiovascular System Exam,ការពិនិត្យប្រព័ន្ធសរសៃឈាមបេះដូង,Cardiovascular System Exam,Cardiovascular System Exam,,,Question,Coded,Acute myocardial infarction;Angina;Pulmonary Oedema;TIA;CVA;Ischemia;Hemorrhagic;Unknown,,,
```

Comment:
@kdaud you might need to add the renamed CVA concept to the coded question otherwise the concept wont load.
![KH-236-rev-1](https://github.com/openmrs/ozone-distro-cambodia/assets/36139599/ca25544d-4479-43df-8d0d-505e1de8910b)

Reply:
I notice other concepts under this section don't seem to be loading as well.